LondonBritishLibrary/orient/BLorient2198.xml

```diff
@@ -44,54 +44,49 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <title ref="LIT3588Miracle"/>
+                                    <note>Numbered as Miracle 4</note>
                                     <incipit xml:lang="gez">፬ተአምሪሃ፡ ለእግዝእትነ፡ ማርያም፡ ድንግል፡ ወላዲተ፡ አምላክ፡ <gap reason="ellipsis"/> ወሀሎ፡ ፩ብእሲ፡ ገብራዊ፡
                                         ዘኮነ፡ ያወትር፡ ሰላመ፡ እግዝእትነ፡ ማርያም፡ ሰርከ፡ ወነግሀ፡ ወመንፈቀ፡ መዓልት፡ </incipit>
                                 </msItem>
                                 <msItem xml:id="ms_i1.4">
                                     <locus from="7r" to="10r"/>
-                                    <title>Fifth miracle</title>
-                                    <note>The healing of the daughter of a brigand by the virtue of the water of the spring which rose under the feet of the
-                                        Child Jesus during the flight into Egypt.</note>
+                                    <title ref="LIT3666Miracle"/>
+                                    <note>Numbered as Miracle 5</note>
                                     <incipit xml:lang="gez">፭ተአምሪሃ፡ ለእግዘትነ፡ ማርያም፡ ድንግል፡ ወላዲተ፡ አምላክ፡ <gap reason="ellipsis"/> ወኮነ፡ እምድኅረ፡ ወላደቶ፡
                                         ለፍቁር፡ ወልዳ፡ ኢየሱስ፡ ክርስቶስ፡ እንዘ፡ ሀለው፡ በውስተ፡ ቤተ፡ ልሔም፡ አዘዘ፡ ሄሮድስ፡ ይቅትሉ፡ ኵሎ፡ ሕፃናተ፡ </incipit>
                                 </msItem>
                                 <msItem xml:id="ms_i1.5">
                                     <locus from="10r" to="11v"/>
-                                    <title>Sixth miracle</title>
-                                    <note>How the Virgin Mary saved the soul of a sinner killed at the 'spring of the Virgin'.</note>
+                                    <title ref="LIT3717Miracle"/>
+                                    <note>Numbered as Miracle 6</note>
                                     <incipit xml:lang="gez">፮ተአምሪሃ፡ ለእግዝእትነ፡ ማርያም፡ ድንግል፡ ወላዲተ፡ አምላክ፡ <gap reason="ellipsis"/> ወእምድኅረ፡ 
                                         አዕር<supplied reason="undefined" resp="PRS8999Strelcyn">ፈ</supplied>ት፡ እግዝእትነ፡ ማርያም፡ መፅአ፡ ፩ፈያታይ፡ ኀበ፡ 
                                         ውእቱ፡ ማይ፡ ዘነቅዐ፡ እምታሕተ፡ እግራ፡ ለማርያም፡ <gap reason="ellipsis"/> ወቀተለ፡ ነፍስ፡ ወትዕፅበ፡ እምብዝኀ፡ ሃፈ፡ ፀሐይ፡ ወኖመ፡ አስተራቲዖ፡
                                         እደዊሁ፡ ወእገሪሁ፡ ኀበ፡ ውእቱ፡ ማይ፡ </incipit>
                                 </msItem>
                                 <msItem xml:id="ms_i1.6">
                                     <locus from="11v" to="13r"/>
-                                    <title>Seventh miracle</title>
-                                    <note>How the Virgin Mary saved the soul of a brigand who killed a girl who put to death the chief of the brigands.
-                                        A double leaf is missing.</note>
+                                    <title ref="LIT3993Miracle">How the Virgin Mary saved the soul of a brigand who killed a girl who put to death the 
```

Comment:
since a page is missing we nees type="incomplete", as mentioned before

Reply:
what do you have against https://pemm.princeton.edu/stories/114 = LIT5183MiracleBrigands?